homepage/views.py

```diff
@@ -14,7 +15,8 @@ def courses(request):
 
 
 def reviews(request):
-    return render(request, 'homepage/reviews/reviews.html')
+    reviews = Review.objects.order_by('-date')

```

Comment:
```suggestion
    reviews = Review.main_feed()
```
Write a function ```def main_feed(cls)``` like in **GUI-TALK-v2** page 29
Finally, you can write a similar test:
![image](https://user-images.githubusercontent.com/57869309/116534893-c755c500-a8eb-11eb-904c-13a2b64cada6.png)

Reply:
@oriAdler gonna change the test data to hold different date in each review (cause otherwise the sort has no meaning), and add this function and test for it